api/src/main/resources/liquibase.xml

```diff
@@ -34,7 +36,7 @@
 			<column name="date_changed" type="datetime">
 				<constraints nullable="true"/>
 			</column>
-			<column name="retired" type="tinyint(1)" defaultValue="0">
```

Comment:
Why not BOOLEAN?

Reply:
liquibase converts boolean to BIT(1) in MySQL ( as found [here](https://stackoverflow.com/questions/16890723/list-all-liquibase-sql-types) ) but according to MySQL standards Boolean is equivalent to Tinyint(1) ( as found [here](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html) )

Reply:
Doesn't openmrs-core already have `retired` fields as BOOLEAN?

Reply:
@dkayiwa  you are right. Just found [this](https://github.com/openmrs/openmrs-core/blob/47463eb05661ec9f8323c96874fc8d6161551389/api/src/main/java/liquibase/ext/datatype/core/MySQLBooleanType.java) where we are explicitly converting Boolean to tinyint(1) in OpenMRS for MySQL. I'll try with boolean now.

Reply:
@dkayiwa , I have made the changes .